tests/authenticity-token.js

```diff
@@ -0,0 +1,37 @@
+var rule = require('../lib/rules/authenticity-token')
+var RuleTester = require('eslint').RuleTester
+
+var ruleTester = new RuleTester()
+
+ruleTester.run("authenticity-token", rule, {
+  valid: [],
+  invalid: [
+    {
+      code: "document.querySelector('form').elements['authenticity_token'].value",
```

Comment:
I have a feeling that `elements.authenticity_token` syntax would still be allowed, since we're only checking string literals?